lib/app.rb

```diff
@@ -0,0 +1,70 @@
+require 'sinatra'
+require 'sidekiq'
+require File.expand_path('../gems.rb', __FILE__)
+require File.expand_path('../workers/dependency.rb', __FILE__)
+require 'yaml'
+require 'redis-objects'
+
+module RubyDepsHelpers
+  def fetch_all(repositories)
+    repositories.each do |gem|
+      DependencyWorker.perform_async(gem.identifier, gem.gemspec_location, gem.gemfile_location)
+    end
+  end
+end
+
+config = <<YAML
+gollum:
+  gollum: true
```

Comment:
What does this signify?

Reply:
I was thinking: top-level is org, next-level is repo -- so `gollum: true` here was meant to mean, "enable the gollum/gollum repo", without setting any custom settings for it.

Reply:
More intuitive formats for settings welcome!

Reply:
With the api_token there this looks a little more intuitive.